apps/example-orchestrator-ui/pages/subscriptions.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {ReactElement, useState} from 'react';
+import {
+    EuiPageTemplate,
+    EuiButton, EuiButtonIcon, EuiDescriptionList, EuiDescriptionListTitle, EuiDescriptionListDescription
+} from '@elastic/eui';
+import {useQuery, gql} from '@apollo/client';
+
+
+const GET_SUBSCRIPTIONS = gql`
+  query GetSubscriptions {
+    subscriptions(first: 1000) {
+    edges {
+      node {
+        note
+        name
+        startDate
+        endDate
+        tag
+        productId
+        portSubscriptionInstanceId
+        vlanRange
```

Comment:
SURF specific? I guess you are implementing our internal API however that version of the api will never be published. I guess I might be missing some discussions over the past week but how will this become generic?

Reply:
If I am not mistaken: there are stories on the sprint: to add GraphQL to orchestrator-core? So when UI 2.0 is ready; we should have generic GraphQL access, right?

For the time being: I want to know what pitfalls we might encounter: and try to tackle Type issues (that arise when other users implement different backends.